### ISSUE TITLE
Checkout to `master` is failing

### ISSUE THREAD

**Issue by arturomf94** [boa-dev/criterion-compare-action#25]
I have a workflow as follows:

```
name: Rust benchmarks

on: [pull_request]
jobs:
  runBenchmark:
    name: run benchmark
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@master
      - uses: jasonwilliams/criterion-compare-action@move_to_actions
        with:
          token: ${{ secrets.GITHUB_TOKEN }}

```

It's failing with:

```
error: pathspec 'master' did not match any file(s) known to git
```

Am I missing anything here or could it be a bug?

**Comment by arturomf94** [comment]
The run can be found [here](https://github.com/OpenMined/sycret/pull/26/checks?check_run_id=3344869703).

**Comment by Razican** [comment]
Why are you using `actions/checkout@master`? I don't think you can use the master branch of the action. The last released version for the checkout action is the v2 I think.

Try changing it for `actions/checkout@v2`

**Comment by arturomf94** [comment]
I tried that but, unfortunately, we get the same error.

**Comment by Razican** [comment]
> I tried that but, unfortunately, we get the same error.

What is the main branch of your repo? By default, the "master" branch is checked out.

**Comment by arturomf94** [comment]
> > I tried that but, unfortunately, we get the same error.
> 
> What is the main branch of your repo? By default, the "master" branch is checked out.

Yes, our main branch is `master`.

**Comment by chipbuster** [comment]
It seems that at some point, the github checkout action stopped doing a fetch (possibly for performance reasons), so that only the PR branch is available locally.

For me, adding a `git fetch` step into the workflow before using this action worked, but I think you could also set `fetch-depth` to zero in the checkout action to fetch full history.

**Comment by Razican** [comment]
I just noticed that the README.md file contains an invalid example. I'll update that and fix other issues and release version 3.0.0.
